server/pbenchinacan/etc/yum.repos.d/pbench.repo

```diff
@@ -0,0 +1,26 @@
+[copr-pbench-test]
+name=Copr pbench repo
+baseurl=https://copr-be.cloud.fedoraproject.org/results/ndokos/pbench-test/epel-8-$basearch
+gpgcheck= 1
+gpgkey=https://copr-be.cloud.fedoraproject.org/results/ndokos/pbench-test/pubkey.gpg
+enabled=1
+enabled_metadata=1
+skip_if_unavailable=1
+
```

Comment:
Do we need more than one repo? If so, why?

Reply:
"Need" is a strong word.  This file has to reference the repos from which the build is going to pull the Server and Dashboard RPMs, respectively.  It's possible that we can do that with fewer than two repos, but I'm not sure.  We can certainly do it with fewer than _three_, but, if we have more than one person doing builds on COPR, we might need (or want) more than one entry.

(I'm open to suggestions as to which ones I should remove and which ones I should leave....)

Reply:
It seems to me that this needs to be a template, rather than a finished repo file:  it is much too ephemeral to keep in the tree as is. Making it a template will complicate the container building script a bit, but not much. And some additional documentation will be needed to allow the user to fill in the gaps.

I guess the main question is: what is this repo (or repos) supposed to provide? That's probably the information that needs to be in the PR.

Reply:
We could make this be, e.g., a jinja2 template, but it's not obvious to me that that is easier for the end-user (i.e., the builder-person) to use than it would be to just edit the file.

I agree that this is highly ephemeral, but I think that a concrete example of a (once-)working file is easier to start from than a template or some description of what the file should look like.

This file is supposed to be an implicit input to `dnf` which allows it to find the repositories for the Dashboard and Server RPMs, respectively.  Unfortunately, that requires at least two pieces of information (the URL for the repo and the URL for the gpgkey), and a name would be nice, as would a custom "section" string.  The rest of each entry is probably boilerplate, but figuring out what to put in those custom fields and getting it in there is (IMO) most easily done by just going in with an editor and touching up what's already there.

But, I'll cut it down to just two entries, and I'll give the sections (and maybe the `name`'s as well) more general and topical values.

Reply:
Instead of providing our own `.repo` file, we could instead use the `dnf config-manager --add-repo` command, if that's more attractive.  This would be invoked in the `container-build.sh` script, and we would still have to get the URLs into the script somehow (but that could be done via an invocation argument, or an environment variable, or we could edit the script or store them some other way).

I think I prefer the `.repo` file approach, though:  for the builder, it's a 'set it and forget it' thing.